frankenphp.go

```diff
@@ -610,6 +626,16 @@ func go_write_headers(rh C.uintptr_t, status C.int, headers *C.zend_llist) {
 		current = current.next
 	}
 
+	handled := false
+	if fc.hooks != nil && fc.hooks.ResponseFilter != nil {
+		handled = fc.hooks.ResponseFilter(int(status), fc.responseWriter.Header(), r)
+	}
+
+	if handled {
```

Comment:
I still fail to understand why we need that.
If I understand correctly https://www.nginx.com/resources/wiki/start/topics/examples/xsendfile/ and https://tn123.org/mod_xsendfile/, this feature is entirely in the scope of the web server and shouldn't leak in this module.

From my understanding, we should let PHP return the full HTTP response, and only after that post-proces it in the Caddy module.

That's why, for instance, Symfony, Laravel, and the like will not generate any response body if this feature is enabled.